nservicebus/upgrades/7to8/index.md

```diff
@@ -17,6 +17,10 @@ NOTE: This is a working document; there is currently no timeline for the release
 
 This document focusses on changes that is affecting general endpoint configuration and message handlers.
 
+## Removed support for .NET Standard
+
+NServiceBus version 8 no longers .NET Standard. Instead, NServiceBus targets .NET Framework 4.5.2 and .NET Core 3.1 directly (read more about our [supported frameworks and platforms](/nservicebus/upgrades/supported-platforms.md)). While this should have no direct impact on endpoint executables, shared message contract assemblies target .NET Standard might need to be adjusted. Refer to the [updating message contracts](message-contracts.md) page for more details.
```

Comment:
This is personal preference (and nitpicky) but I prefer not to word things from a "point in time" perspective, even in the upgrade guides. In practice, this means avoiding things like "no longer" or "anymore". Reason being that we'll eventually release version 9 in a couple of years and people will still need to upgrade to v8 and this guide, while still accurate, won't be current.

```suggestion
NServiceBus version 8 does not .NET Standard. Instead, NServiceBus targets .NET Framework 4.5.2 and .NET Core 3.1 directly (read more about our [supported frameworks and platforms](/nservicebus/upgrades/supported-platforms.md)). While this should have no direct impact on endpoint executables, shared message contract assemblies that target .NET Standard might need to be adjusted. Refer to the [updating message contracts](message-contracts.md) page for more details.
```

or

```suggestion
.NET Standard support was removed in NServiceBus version 8. Instead, NServiceBus targets .NET Framework 4.5.2 and .NET Core 3.1 directly (read more about our [supported frameworks and platforms](/nservicebus/upgrades/supported-platforms.md)). While this should have no direct impact on endpoint executables, shared message contract assemblies that target .NET Standard might need to be adjusted. Refer to the [updating message contracts](message-contracts.md) page for more details.
```